README.md

```diff
@@ -1,63 +1,64 @@
+Version control is _\[both\]_ a system and practice for managing changes in a _\[centralized space that allows contributors to access, modify and contribute\]_ file or digital content _\[simultaneously\]_. By using a set of version control practices and tools, one can revert to a previous state of a project and view or manipulate its content. These are crucial tools that ensure that play a big part in ensuring the quality of the final product.
 
-Version control systems provide a centralized space for the project which allows contributors to access, modify and contribute simultaneously.
+~~Version control systems provide a centralized space for the project which allows contributors to access, modify and contribute simultaneously.~~
 
 In fast-paced development environments, these systems make it easier for teams to work efficiently. Most often, these systems are designed to record every modification to the contents of the project. When there is an issue, those working on the project can compare previous versions to fix issues, errors, or inconsistencies with minimal disruption.
  (__Source__:  [Altassian](https://www.atlassian.com/git/tutorials/what-is-version-control)).
 
-![Version Control](Version_Control.webp)
+![Version Control](Version_Control.webp) 
 __Image Source:__ [Simplilearn](https://www.simplilearn.com/ice9/free_resources_article_thumb/Version_Control.png)
 
 ## Source Code Version Control
 Source-code version control is a tool that is specifically used to manage code files. 
 
 - It keeps track of the history of code changes, including the author and time of the modifications. 
 - It allows the creation of multiple versions of the code, each in its branch, to enable teams to work on different features without disrupting the main code branch.
-- The tool also enables merging changes made in separate branches back into the main code branch and resolves any conflicts that arise during this process (__Source__:  [Open AI](https://chat.openai.com)).
+- The tool also enables merging changes made in separate _\[development\]_ branches back into the main code branch and resolves any conflicts that arise during this process (__Source__:  [Open AI](https://chat.openai.com)).
 
 Today, there are multiple source-code version control tools available on the market. Some of the most popular ones are Git, Subversion, and Mercurial. Each of these has its own unique operations, rules, and advantages (__Source__:  [Open AI](https://chat.openai.com)).
 
-The nature of the project one is working on really defines which version control system is best for that specific project's needs. However, in this article, we will be focusing on the GitHub environment.
+The nature of the project one is working on really defines which version control system is best for that specific project's needs ([ref?]()). However, in this article, we will be focusing on the GitHub environment.
```

Comment:
The first sentence is a curious statement. What aspect of the _nature_ of a project determines the choice of VCS? If you can't explain why then you should have a source that I can look at that will provide the explanation. If you can't explain, and you don't have a source to support this claim, then the sentence should be removed.